book/src/workflow/analysis.md

```diff
@@ -39,8 +41,6 @@ linked with the question "which things can I predict given which other things?"
 
 In python, we can generate a plotly heatmap of *dependence probability*.
 
-<p class="multilang">
```

Comment:
Should this have been replaced with a `<div class="tabbed-blocks">`, just to be consistent?